<launch>
    <node name="open_loop"      pkg="uranus_dp" type="open_loop"/>
    <node name="dummy_joystick" pkg="uranus_dp" type="dummy_joystick"/>
    <node name="allocator"      pkg="uranus_dp" type="allocator" output="screen"/>
</launch>